inventory/group_vars/br.yml

```
@@ -0,0 +1,22 @@
+---
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Brazil
+country_code: BR
+currency: BRL
+
+locale: pt_BR # Default locale for OFN instance
+available_locales: pt_BR # Available locales for OFN instance
+language: pt_BR.UTF-8 # Default language for the server
+language_packages: # Language packages for the server
+  - language-pack-en-base
+l10n_repo: https://github.com/openfoodbrazil/l10n_br.git
```

I have imported l10n_br to ofn :tada: 
you can now switch this line:
```suggestion
l10n_repo: https://github.com/openfoodfoundation/l10n_br.git
```

---

This change should be good now that https://github.com/openfoodfoundation/l10n_br.git has been added :+1: